microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -226,7 +227,8 @@ def _setup_node(self) -> None:
         # which breaks when another tool checks for it's existence before building...
         # like cmake, meson, make, autoconf, etc.
         self._node.tools[Ninja].install()
-        self._node.tools[Pip].install_packages("pyelftools")
+        if not isinstance(self._os, Debian):
+            self._node.tools[Pip].install_packages("pyelftools")
```

Comment:
Will Debian pass without this package?

Reply:
This is a good comment, the install order of pyelftools turns out to be important. It's listed as a dependency in a few places, but this is kind of confusing. I'll make this more consistent so we're not relying on which tool runs first and possibly trying to install it multiple times.